internal/cmd/rsl/rsl.go

```diff
@@ -12,8 +13,9 @@ func New() *cobra.Command {
 		Short: "Tools to manage the repository's reference state log",
 	}
 
-	cmd.AddCommand(record.New())
 	cmd.AddCommand(annotate.New())
+	cmd.AddCommand(check.New())
```

Comment:
My gut feeling is we may want to move remote operations under its own subgroup - e.g. `gittuf rsl remote check`. wdyt?

Reply:
That sounds fine to me.